### PR TITLE
New Feature: nocolor configuration flag

### PR DESCRIPTION
**Pull request by mrcrilly**
A simple commit to add a "nocolor" option.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -254,6 +254,14 @@ gopass
 └── 0xB5B44266A3683834 - Gopher <gopher@golang.org>
 ```
 
+### Debugging
+
+To debug `gopass`, set the environment variable `GOPASS_DEBUG` to `true`.
+
+### Disabling Colors
+
+Disabling colours is as simple as `gopass config nocolors true`.
```
Comment (dominikschulz) [inline README.md]: There is tiny typo in here: It's supposed to be "nocolor" instead of "nocolors".

Comment (mrcrilly) [inline README.md]: Sorry about the delay. I'll get to this soon.

**Comment by mrcrilly** [comment]
Updated README to resolve typo (I forgot you could edit stuff in GitHub directly these days - sorry.)

**Comment by mrcrilly** [comment]
What's this now waiting for?

**Pull request merged**